### DISCUSSION TITLE
Toggle Background For Active Space

### DISCUSSION THREAD

**Discussion by ssate** [FelixKratz/SketchyBar#83] [Q&A] [answered]
Hi,

I was wondering whether it is possible to toggle the background for the current active / selected space (this is probably very simple and I am missing something...)

I have tried the `background.drawing=toggle` property, though, this alone seems to permanently enable the background.

Thanks in advance!

**Comment by FelixKratz** [comment, marked as answer]
You can achieve that by supplying a custom space script like this:
```bash
#!/usr/bin/env bash

if [ "$SELECTED" = "true" ]; then
  sketchybar -m --set $NAME icon.highlight=on background.drawing=on
else
  sketchybar -m --set $NAME icon.highlight=off background.drawing=off
fi
```
which you can save as ~/.config/sketchybar/plugins/space.sh
and then simply add it as the script for all of your space components:
```bash
sketchybar -m --set <space_name> script="~/.config/sketchybar/plugins/space.sh"
```

**Reply by ssate** [reply]
awesome thanks so much!
